.travis.yml

```diff
@@ -133,5 +129,5 @@ jobs:
         OSX_SDK=10.11
         RUN_UNIT_TESTS=false
         RUN_FUNCTIONAL_TESTS=false
-        GOAL="all deploy"
+        GOAL="deploy"
```

Comment:
Why is this change needed? It doesn't seem like the mac cross build should be affected at all?

Reply:
The origin motivation of 3d69853 is to vompile all binaries, not only qt. But after cf01fd6 has been merged, it would build all binaries even when the target is deploy.

Reply:
@MarcoFalke Do you think that this change is unnecessary?